src/App.tsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react';
+import React from "react";
```

Comment:
I'm trying to figure out why you're using sometimes single quotes and sometimes double ones :thinking:

Reply:
Forgot to format